Governance/manage/manage-intro.md

```diff
@@ -0,0 +1,30 @@
+# Guidelines to manage climate dataset
+
+These guidelines are aimed to dataset managers and cover all aspects of managing an authoritative or a replicated dataset on a local server.
+Some of examples we show here are NCI based as this is where we managed most of our data. Hopefully the examples can still be extrapolated to other platforms. 
+data custodians and stewards in technical data management at NCI:
+
+Project handling 
+  writers group creation
+  use of posix permissions
+  use of ACLs
```

Comment:
Has "ACL" been defined prior to this point? I don't think it has but it's a bit hard to think about in commit view. All of these things need to be defined briefly before we talk about them which leads to a bit of a chicken and egg problem, so I'm not going to say we NEED to define them here but also this might be way too jargon heavy for a new data custodian?

Reply:
They were only there as a reminder to cover them, there's a proper introduction, and acts don't get mentioned until later in the manage-project page when we talk about writers' groups. Then there's a link to the relevant page in the technical guide.